src/components/endpoint-selector/index.js

```diff
@@ -16,14 +16,13 @@ class EndpointSelector extends Component {
 	}
 
 	componentDidMount() {
-		if ( ! this.props.endpoints.length && this.props.version ) {
+		if ( this.props.version ) {
 			this.props.loadEndpoints( this.props.api, this.props.version );
```

Comment:
I think this is not the right place to call `loadEndpoints`.  Instead we should do it when something changes in the API selector or the version selector.  This way, the endpoints will probably be loaded already when we go to select one, and also there will be a lot less network requests to `/help` or `/wp/v2?context=help`.